src/aap_eda/core/models/organization.py

```diff
@@ -0,0 +1,26 @@
+from ansible_base.lib.abstract_models.organization import AbstractOrganization
+from django.conf import settings
+from django.db import models
+
+
+class OrganizationManager(models.Manager):
+    def get_default(self):
+        return self.get(name=settings.DEFAULT_ORGANIZATION_NAME)
+
+
+class Organization(AbstractOrganization):
```

Comment:
@Dostonbek1 Shouldn't these models be coming from DAB. We install the app ansible_base.rbac shouldn't that app be installing the models needed by the DAB app.

Reply:
`dab_rbac` (ansible_base.rbac as import path) is an app, but it's agnostic to the organization model. It allows defining parent models of other models so that roles can delegate permission to child objects, and making organizations a common parent is just what's usually done.

The `Team` model (analogous to `auth.Group`) is special, in that it has RBAC implications, but it is not a concrete model inside of the rbac app. The project using DAB points DAB to its own team model in settings, and rbac app migrations point to it as a swappable model. This is similar to (and modeled after) the `User` model. The models in DAB RBAC have relational connections to both user & team models, but don't define these models.

Reply:
Is every component AAP, EDA going to have their own Organization, Team concepts? I thought that was a shared concept across the platform, and if a change needs to be made it would be made in the shared model instead of each app creating their own models, leading to customer confusion about these concepts. In EDA we don't have a requirement for a custom Organization and Team, we can use the same concrete model that AAP uses,  which would be consistent across the platform. If we look at the django.contrib.auth it does provide a concrete User model that apps can use but if they need to customize it you can inherit from the AbstractUser and build your own models. I was under the impression that DAB would be providing a similar capability with RBAC support.

Reply:
I'll volunteer the background that AWX already uses the `auth.User` model, and galaxy_ng already uses the `auth.Group` model. Earlier iterations of this RBAC library used `auth.Permission` but then we reversed course and added a concrete permission model in the dab_rbac app, due to similar arguments to the ones you are making here.

Reply:
We can have a long term goal of a shared implementation, but we need a plan to get there over time.  Moving everything to DAB is not feasible in the short term.

Reply:
First we need to list the requirements, fields, and behaviors for each model in each component and come up with a union of those for the general case.   We might need to split out certain fields and behaviors into other models if they don't really fit into the general one that works for the other components.    Then we need to plan when those changes can happen over several releases and the migration path for each one.

Reply:
The cost of that effort is much greater than having a concrete implementation in each component.   The concrete implementation helps us define the requirements for that component anyway.  Defining the concrete requirements is part of the abstraction/generalization process.